sys/arm64/conf/GENERIC-MORELLO-BENCHMARKABI

```diff
@@ -0,0 +1,25 @@
+#
+# GENERIC-MORELLO-BENCHMARKABI -- Generic kernel configuration file for CheriBSD/Morello benchmark ABI kernel
+#
+# For more information on this file, please read the config(5) manual page,
+# and/or the handbook section on Kernel Configuration Files:
+#
+#    https://www.FreeBSD.org/doc/en_US.ISO8859-1/books/handbook/kernelconfig-config.html
+#
+# The handbook is also available locally in /usr/share/doc/handbook
+# if you've installed the doc distribution, otherwise always see the
+# FreeBSD World Wide Web server (https://www.FreeBSD.org/) for the
+# latest information.
+#
+# An exhaustive list of options and more detailed explanations of the
+# device lines is also present in the ../../conf/NOTES and NOTES files.
+# If you are in doubt as to the purpose or necessity of a line, check first
+# in NOTES.
+#
+# $FreeBSD$
```

Comment:
These are useless and mostly gone upstream, we should purge them from our local configs.

Reply:
Ok, I'll avoid adding more here then.